src/main/java/uk/gov/digital/ho/hocs/info/api/CorrespondentTypeService.java

```diff
@@ -38,6 +38,13 @@ public CorrespondentTypeService(CorrespondentTypeRepository correspondentTypeRep
         return correspondentTypes;
     }
 
+    Set<CorrespondentType> getSelectableCorrespondentTypesByCaseType(String caseType) {
+        log.debug("Getting all correspondent types for Case Type {}", caseType);
```

Comment:
Here we are only getting the selectable correspondent types rather than all correspondent types

```suggestion
        log.debug("Getting all selectable correspondent types for Case Type {}", caseType);
```